import/popups/PopupOpener.qml

```diff
@@ -82,47 +98,18 @@ Timer {
         var winId = data.winId
         switch (topic) {
         case "embed:alert": {
-            if (pageStack.busy) {
-                // Wait until pagestack ready
-                root.data = data
-                root.topic = topic;
-                pageStack.busyChanged.connect(busyChanged)
-            } else {
-                alert(data)
-            }
+            popupOpen(alert, data)
```

Comment:
It's just a thought, but since `data` doesn't change, you could even simplify the logic by binding the data directly to the function at this point (something like `popupOpen(alert.bind(data))`, but this is untested).

Reply:
I dropped this commit. Since customizable popus was merged, this change is unnecessary.

Reply:
Ah, okay. Is there a replacement commit?

Reply:
Yes. I that's it.

Reply:
I couldn't resist improving `PopupOpener.qml` a bit in the last commit.